src/main/java/ro/vladutit/Don/t/forget/v2/controller/CategoryController.java

```diff
@@ -15,15 +15,14 @@
     @Autowired
     private CategoryService categoryService;
 
-    //add new category with a form
+    //add new category
```

Comment:
```suggestion
    // Add new category
```

Reply:
I have made these changes everywhere. I made an update to commit (--amend). Please take a look. Thank you.